foyer/forcefields/xml/oplsaa.xml

```diff
@@ -174,25 +174,25 @@
   <Type name="opls_168" class="HO" element="H" mass="1.008" def="H[O;%opls_167]" overrides="opls_155" doi="10.1021/ja9621760"/>
   <Type name="opls_169" class="opls_169" element="O" mass="15.9994"/>
   <Type name="opls_170" class="opls_170" element="H" mass="1.008"/>
-  <Type name="opls_171" class="opls_171" element="O" mass="15.9994"/>
-  <Type name="opls_172" class="opls_172" element="H" mass="1.008"/>
-  <Type name="opls_173" class="opls_173" element="C" mass="12.011"/>
-  <Type name="opls_174" class="opls_174" element="C" mass="12.011"/>
+  <Type name="opls_171" class="OH" element="O" mass="15.9994" def="[O;X2]([%opls_173,%opls_174])(H)" desc="O in triols" overrides="opls_154" doi="10.1021/ct200731v"/>
+  <Type name="opls_172" class="HO" element="H" mass="1.008" def="[H;X1]([O;%opls_171])" desc="H(O) in triols" overrides="opls_155" doi="10.1021/ct200731v"/>
+  <Type name="opls_173" class="CT" element="C" mass="12.011" def="[C;X4]([H])([H])([O;X2]([H]))([C;X4]([O;X2]([H])))" desc="C(H2)(OH) in triols" overrides="opls_157, opls_158" doi="10.1021/ct200731v"/>
```

Comment:
This might need to be more specific to match to a triol

<img width="521" height="571" alt="image" src="https://github.com/user-attachments/assets/218baab8-3735-4fb9-91fd-41ce39db8f49" />

Would match diols no problem. Idk what the solution is though since there is no specific case for where the three alcohol groups are relative to the carbon other than that one is attached directly to opls_173. 

Is the main molecule propane-1,2,3-triol?
In that case, it could be like 
`def="[C;X4]([H])([H])([O;X2]([H]))([C;X4]([O;X2]([H]))([C;X4]([O;X2]([H]))))"`
Which I think covers the terminal CH2OH of this molecule.
<img width="331" height="258" alt="image" src="https://github.com/user-attachments/assets/f534fd90-8672-4467-8175-6dd69ce85f43" />